src/services/car/messageLead.ts

```diff
@@ -245,3 +246,75 @@ export const resendMessageLead = async ({
     result: {},
   }
 }
+
+export const fetchDealerWhatsAppLeads = async ({
```

Comment:
this endpoint is on the reporting service, but the file is located in the `car/service/messageLead`, so it seems to be misplaced

Reply:
I saw that we have all other endpoint there (e.g fetchDealerMessageLeads or fetchDealerCallLeads), do you want me to add add those endpoint into separate files?

Reply:
uhmm the convention would be to structure the methods / files / folders by the api structure. let's keep what we have now and clean up separately / globally